Tasks/AzureRmWebAppDeployment/task.json

```diff
@@ -402,7 +402,7 @@
         "Failedtodeploywebapppackageusingkuduservice": "Failed to deploy App Service package using kudu service : %s",
         "Unabletodeploywebappresponsecode": "Unable to deploy App Service due to error code : %s",
         "MSDeploygeneratedpackageareonlysupportedforWindowsplatform": "MSDeploy generated packages are only supported for Windows platform.",
-        "UnsupportedinstalledversionfoundforMSDeployversionshouldbealteast3orabove": "Unsupported installed version: %s found for MSDeploy. version should be alteast 3 or above",
+        "UnsupportedinstalledversionfoundforMSDeployversionshouldbeatleast3orabove": "Unsupported installed version: %s found for MSDeploy. version should be at least 3 or above",
```

Comment:
minor: other places you used "atleast" while here it is "at least" for the same string.